### PR TITLE
tests: less echo hello

### PR DESCRIPTION
**Pull request by dankenigsberg**
Use `libvmi` to simplify a few tests. This makes the tests slightly more readable, dropping `echo hello` boilerplate when defining Cirros VMs.

```release-note
NONE
```

### CODE REVIEW

File: tests/dryrun_test.go [hunk 1]
```diff
@@ -61,7 +61,8 @@ var _ = Describe("[sig-compute]Dry-Run requests", func() {
 		resource := "virtualmachineinstances"
 
 		BeforeEach(func() {
-			vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+			vmi = libvmi.NewAlpine()
+			vmi.ObjectMeta.Namespace = util.NamespaceTestDefault
```
Comment (xpivarc) [inline tests/dryrun_test.go:65]: This should be not required. Can you please file a bug/issue?

File: tests/vm_test.go [hunk 2]
```diff
@@ -2132,7 +2121,8 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 
 	Context("crash loop backoff", func() {
 		It("should backoff attempting to create a new VMI when 'runStrategy: Always' during crash loop.", func() {
-			vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), "echo Hi\n")
+			vmi := libvmi.NewCirros()
+			vmi.Namespace = util.NamespaceTestDefault
```
Comment (xpivarc) [inline tests/vm_test.go:2125]: Same as above

File: tests/dryrun_test.go (hunk 1 above)
Comment (dankenigsberg) [reply tests/dryrun_test.go:65]: I wondered if someone would call me out about this. With your permission, I'll take these files' cleanup to a follow up PR.

Comment (xpivarc) [reply tests/dryrun_test.go]: I know you are not introducing anything new but please consider including this change in this PR.
It should be a simple replacement, for example, replace all `vmi.Namespace` with `util.NamespaceTestDefault` and so on.

**Review by xpivarc — COMMENTED** [review]
/approve

nit: https://github.com/kubevirt/kubevirt/pull/8696/commits/24c1b9bd99ac7a6f407a34ca535c541458187fab 
"Two tests seem to use vmis that are a bit more complex to  define with
libvmi, so fixing them is postponed to a future PR." I am not if this is helpful without specifying which 2 are left out. (or why)

**Comment by 0xFelix** [comment]
No objections, except that I have to agree with @xpivarc's nit

Putting it on hold, feel free to fix or just cancel the hold

/lgtm
/hold

**Comment by 0xFelix** [comment]
@dankenigsberg Yes, thank you.

/lgtm

**Pull request merged**